AudioPlayerRecorderLibrary/build.gradle

```diff
@@ -1,12 +1,12 @@
-apply plugin: 'android-library'
+apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 19
-    buildToolsVersion "21.1.1"
+    compileSdkVersion 23
+    buildToolsVersion "23.0.1"
 
     defaultConfig {
         minSdkVersion 8
-        targetSdkVersion 19
+        targetSdkVersion 23
         versionCode 1
         versionName "1.0"
```

Comment:
`versionCode` and `versionName` should be updated given the behavior is slightly changed. I'd suggest `versionCode 2` and `versionName 1.1`.